src/data/vagrant.json

```diff
@@ -74,5 +74,43 @@
 			}
 		]
 	},
-	"navigationHeaderItems": {}
+	"navigationHeaderItems": {
+		"documentation": [
+			{
+				"icon": "docs",
+				"label": "Introduction",
+				"pathSuffix": "docs"
+			},
+			{
+				"icon": "docs",
+				"label": "General Documentation",
+				"pathSuffix": "docs"
+			},
+			{
+				"icon": "docs",
+				"label": "Vagrant Cloud",
+				"pathSuffix": "cloud"
+			}
+		]
+	},
+	"rootDocsPaths": [
+		{
+			"iconName": "docs",
+			"name": "Introduction",
+			"path": "intro",
+			"shortName": "Intro"
+		},
+		{
+			"iconName": "docs",
+			"name": "General Documentation",
+			"path": "docs",
+			"shortName": "Documentation"
+		},
```

Comment:
For the sidebar on /vagrant, we need to list this General Documentation item first.

Reply:
Cool, will update!

Reply:
I just realized that `rootDocsPaths` was already defined in the data file 😶‍🌫️ lol so I removed it from this PR!